.aicoe-ci.yaml

```diff
@@ -0,0 +1,17 @@
+check:
+  - thoth-precommit
+  - thoth-build
+build:
+  base-image: "registry.access.redhat.com/ubi8"
+  build-stratergy: Dockerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
+  dockerfile-path: Dockerfile
+  custom_tag: latest # custom tag to be push to registry
```

Comment:
Also, do I let this be 'latest'?

Reply:
we normally keep only tag version, but we can have the kebechet latest tag as well.

Reply:
So just remove custom_tag key?

Reply:
Thanks I have removed it